frontend/app/routes/$lang+/_public+/apply+/$id+/confirmation.tsx

```diff
@@ -155,7 +155,7 @@ export default function ApplyFlowConfirm() {
 
   const mscaLink = <InlineLink to={t('confirm.msca-link')} />;
   const dentalContactUsLink = <InlineLink to={t('confirm.dental-link')} />;
-  const cdcpLink = <InlineLink to={'/status'} />;
+  const cdcpLink = <InlineLink to={`/${i18n.language}/status`} />;
```

Comment:
This is not the correct way to link to internal pages anymore. Do this instead:

``` tsx
<InlineLink routeId="$lang+/_public+/status+/index" params={params} />
```

Where `params` is initialized via `useParams()`.

This is because all internal paths are now bilingual, and we need to link to the correct one depending on the current language.